grml-debootstrap

```diff
@@ -1687,30 +1696,20 @@ iface eth0 inet dhcp
 "
 
   if [ -n "$NOINTERFACES" ] ; then
-    einfo "Not installing /etc/network/interfaces as requested via --nointerfaces option" ; eend 0
+    einfo "Not installing /etc/systemd/network/80-dhcp.network as requested via --nointerfaces option"
+    eend 0
   elif [ -n "$USE_DEFAULT_INTERFACES" ] ; then
-    einfo "Installing default /etc/network/interfaces as requested via --defaultinterfaces options."
+    einfo "Installing default /etc/systemd/network/80-dhcp.network as requested via --defaultinterfaces options."
+    echo "$DEFAULT_SYSTEMD_NETWORKD" > "${MNTPOINT}/etc/systemd/network/80-dhcp.network"
     echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
     eend $?
   elif [ -n "$VIRTUAL" ] ; then
-    einfo "Setting up Virtual Machine, installing default /etc/network/interfaces"
-    echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
-    eend $?
-  elif [ -r /etc/network/interfaces ] ; then
-    einfo "Copying /etc/network/interfaces from host to target system"
-    cp $VERBOSE /etc/network/interfaces "${MNTPOINT}/etc/network/interfaces"
-    eend $?
-  else
-    ewarn "Couldn't read /etc/network/interfaces, installing default /etc/network/interfaces"
+    einfo "Setting up Virtual Machine, installing default /etc/systemd/network/80-dhcp.network"
+    echo "$DEFAULT_SYSTEMD_NETWORKD" > "${MNTPOINT}/etc/systemd/80-dhcp.network"
     echo "$DEFAULT_INTERFACES" > "${MNTPOINT}/etc/network/interfaces"
     eend $?
   fi
 
-  # install config file providing some example entries
-  if [ -r /etc/network/interfaces.examples ] && [ ! -r "$MNTPOINT/etc/network/interfaces.examples" ] ; then
-     cp /etc/network/interfaces.examples "$MNTPOINT/etc/network/interfaces.examples"
-  fi
-
   eend 0
```

Comment:
Please mention in the commit message that we also don't take over /e/n/interfaces.examples, if it exists.

Reply:
Ack.